### ISSUE TITLE
help me it's only fetching the first 20 products.

### ISSUE THREAD

**Issue by andy3o0** [notrab/headless-dropshipping-starter#66]
help me it's only fetching the first 20 products.

**Comment by andy3o0** [comment]
Kindly help I need to deliver it to the client :(

**Comment by notrab** [comment]
Hey @andy3o0 

Do you see anything in the response? I'd take a look at the Printful API docs to see if they accept a pagination param that you can adjust.

**Comment by notrab** [comment]
I will try to replicate this and support further. In the meantime I’d suggest looking at the products that come back from printful and see if the products are being paginated correctly inside the code for the index page props.

**Comment by JBenAnderson** [comment]
Hi first off great project and resource thanks @notrab!  I see no one has responded to this issue yet and wanted to say that I ran into this same problem with only returning 20 products and found a fix.  

I was able to increase the size by adding a parameter within the printful-request get statement near the bottom of index.tsx to set limit to 100 (which I believe is the max amount not sure).  This will return up to 100 products on your page at a time.  

<img width="638" alt="Screen Shot 2022-07-31 at 5 10 59 PM" src="https://user-images.githubusercontent.com/62675858/182045501-5afa56d0-5665-451c-9f69-a8a172e64507.png">
